pkg/pow/miningTasker.go

```diff
@@ -132,13 +136,29 @@ func (mt *MiningTasker) GetWork(chan *Work) (*Work, bool) {
 		m2, err := mt.proxy.BatchGet([]string{valKey})
 		if err != nil {
 			mt.log.Info("Could not retrieve pricing data for current request id: %v", err)
-			return nil, false
+			//return nil, false
```

Comment:
Why would it continue with an err here? If the db doesn't return even one of the values a solution can't be  submitted right?

Reply:
I agree, I think it should return, I hit this a lot because of #229